ocw/lib/EC2.py

```diff
@@ -209,66 +196,13 @@ def delete_all_clusters(self):
-                              -(?P<flavor>EC2)
-                              \.
-                              (?P<arch>[^-]+)
-                              -
-                              (?P<kiwi>\d+\.\d+\.\d+)
-                              -
-                              (?P<type>(BYOS|On-Demand))
-                              -Build
-                              (?P<build>\d+\.\d+)
-                              \.raw\.xz
-                              ''', re.RegexFlag.X),
-            # openqa-SLES15-SP2.x86_64-0.9.3-EC2-HVM-Build1.10.raw.xz'
-            # openqa-SLES15-SP2-BYOS.x86_64-0.9.3-EC2-HVM-Build1.10.raw.xz'
-            # openqa-SLES15-SP2.aarch64-0.9.3-EC2-HVM-Build1.49.raw.xz'
-            # openqa-SLES15-SP4-SAP-BYOS.x86_64-0.9.3-EC2-Build150400.1.31.raw.xz
-            re.compile(r'''^openqa-SLES
-                              (?P<version>\d+(-SP\d+)?)
-                              (-(?P<type>[^\.]+))?
-                              \.
-                              (?P<arch>[^-]+)
-                              -
-                              (?P<kiwi>\d+\.\d+\.\d+)
-                              -
-                              (?P<flavor>EC2[-\w]*)
-                              -Build(\d+\.)?
-                              (?P<build>\d+\.\d+)
-                              \.raw\.xz
-                              ''', re.RegexFlag.X),
-            # openqa-SLES12-SP4-EC2-HVM-BYOS.x86_64-0.9.2-Build2.56.raw.xz'
-            re.compile(r'''^openqa-SLES
-                              (?P<version>\d+(-SP\d+)?)
-                              -
-                              (?P<flavor>EC2[^\.]+)
-                              \.
-                              (?P<arch>[^-]+)
-                              -
-                              (?P<kiwi>\d+\.\d+\.\d+)
-                              -
-                              Build
-                              (?P<build>\d+\.\d+)
-                              \.raw\.xz
-                              ''', re.RegexFlag.X)
-        ]
-        return self.parse_image_name_helper(img_name, regexes)
-
     def cleanup_all(self):
-        cleanup_ec2_max_snapshot_age_days = PCWConfig.get_feature_property('cleanup', 'ec2-max-snapshot-age-days',
-                                                                           self._namespace)
-        cleanup_ec2_max_volumes_age_days = PCWConfig.get_feature_property('cleanup', 'ec2-max-volumes-age-days',
-                                                                          self._namespace)
-        self.cleanup_images()
-        if cleanup_ec2_max_snapshot_age_days >= 0:
-            self.cleanup_snapshots(cleanup_ec2_max_snapshot_age_days)
-        if cleanup_ec2_max_volumes_age_days >= 0:
-            self.cleanup_volumes(cleanup_ec2_max_volumes_age_days)
+        cleanup_ec2_max_age_days = PCWConfig.get_feature_property('cleanup', 'ec2-max-age-days', self._namespace)
+
+        if cleanup_ec2_max_age_days >= 0:
```

Comment:
```suggestion
        if cleanup_ec2_max_age_days > 0:
```

I don't think that setting this value to `0` makes any sense except for disabling this check or am I wrong? Is `0` a valid setting?

Reply:
fixed